### PR TITLE
Contact Us section 

### PR DESCRIPTION
**Pull request by keshavlikhar**
Add Contact Us section in Index Page

### CODE REVIEW

**Comment by mooocer** [comment]
@harshitlikhar the `.idea` folder is about the IDE settings. It's not meant to be pushed on the repo. You can use `git reset --soft HEAD` to remove your commit and re-commit your changes without the `.idea` folder.

File: index.html [hunk 1]
```diff
@@ -218,6 +217,25 @@ <h1>hackSocIndore</h1>
     </div>
     <!-- End Culture -->
 
+    <!--========== Contact Us ==========-->
+
+    <div style="background-color: lightgray; padding-top: 50px; padding-bottom: 60px;">
+
+        <div class="section section--xTall">
+            <div class="wrapper wrapper--thin">
```
Comment (mooocer) [inline index.html]: do we have `wrapper`, `wrapper--thin`, `section`, and `section--xTall` classes?

**Review by mooocer — COMMENTED** [review]
It looks nice @harshitlikhar ! :tada: Just review my few comments on the code. 
And do you think you can make css classes for the inline css you have applied?

File: index.html (hunk 1 above)
Comment (keshavlikhar) [reply index.html]: I don't think they are needed , should remove them

File: index.html [hunk 2]
```diff
@@ -218,6 +217,21 @@ <h1>hackSocIndore</h1>
     </div>
     <!-- End Culture -->
 
+    <!--========== Contact Us ==========-->
+
+    <div style="background-color: lightgray; padding-top: 50px; padding-bottom: 60px;">
+                <div class="text-group text-center">
```
Comment (mooocer) [inline index.html]: Please fix the Indentation herei

File: index_mobile.html [hunk 3]
```diff
@@ -199,6 +199,19 @@ <h1>hackSocIndore</h1>
 </div>
 <!-- End Culture -->
 
+<!--========== Contact Us ==========-->
+
+<div style="background-color: lightgray; padding-top: 50px; padding-bottom: 60px;">
+            <div class="text-group text-center">
```
Comment (mooocer) [inline index_mobile.html]: indentation

File: index_mobile.html [hunk 4]
```diff
@@ -199,6 +199,19 @@ <h1>hackSocIndore</h1>
...
+                <h3 class="h2">Contact Us</h3>
+                <p style="color: dimgrey; padding-bottom: 20px" >For any queries.</p>
+            </div>
+
+            <div style="text-align: center;">
```
Comment (mooocer) [inline index_mobile.html]: indentation

**Review by mooocer — CHANGES_REQUESTED** [review]

**Comment by mooocer** [comment]
Nice work Keshav! :tada: :clap:

**Pull request merged**